cordova-lib/src/hooks/HooksRunner.js

```diff
@@ -87,6 +93,12 @@ module.exports = HooksRunner;
  */
 module.exports.fire = globalFire;
 function globalFire(hook, opts) {
+    if (isHookDisabled(opts, hook)) {
+        /* jshint -W064 */
```

Comment:
Just a minor nit: do we need for this ignores? We don't have neither 'strict mode' nor JSHint `newcap` option enabled, so this is probably just a no-op.